tshirts.cpp

```diff
@@ -12,11 +12,15 @@ char size(int cms) {
     }
     return sizeName;
 }
+void sizetest(int cms, char sizename) {
+    assert(size(cms) == sizename);
+}
 
 int main() {
-    assert(size(37) == 'S');
-    assert(size(40) == 'M');
-    assert(size(43) == 'L');
+    sizetest(37, 'S');
+    sizetest(40, 'M');
+    sizetest(43, 'L');
+    sizetest(38, 'S');

```

Comment:
Why is 38 expected to be `S` and not `M`?
How would this behavior affect the user?